poetry.lock

```diff
@@ -2159,14 +2159,14 @@ resolved_reference = "6c96e77db3b9530060ba1796f46e1481d7c4dd98"
 
 [[package]]
 name = "pytest-timeout"
-version = "1.4.2"
-description = "py.test plugin to abort hanging tests"
+version = "2.1.0"
```

Comment:
Please revert all the changes in the lock file to the version that is currently in main